web/src/components/dateTime.js

```diff
@@ -10,25 +9,40 @@ class Datetime extends Component {
     };
   }
   render() {
-    const date = new Date(this.state.schedule.startDateTime);
-    const month = date.toLocaleString('US', {month: 'long'}).substring(0, 3);
-    const year = date.toLocaleString('US', {year: 'numeric'});
-    const day = date.toLocaleString('US', {day: '2-digit'});
+    const startDate = new Date(this.state.schedule.startDateTime);
+    const endDate = new Date(this.state.schedule.endDateTime);
+    const month = startDate.toLocaleString('US', {month: 'long'}).
+        substring(0, 3);
+    const day = startDate.toLocaleString('US', {day: 'numeric'});
     const options = {
       hour: '2-digit',
       minute: '2-digit',
     };
-    const time = date.toLocaleString('US', options);
+    const startTime = startDate.toLocaleString('US', options);
+    const endTime = endDate.toLocaleString('US', options);
+    let dateForm = '';
+    if (day >= 11 && day <= 13) {
+      dateForm = 'th';
+    } else {
+      switch (day % 10) {
+        case 1: dateForm = 'st';
+        case 2: dateForm = 'nd';
+        case 3: dateForm = 'rd';
+        default: dateForm = 'th';
+      }
+    }
+
     return (
-      <Typography variant='body2' style={{marginBottom: 5}}>
-        {month} {day}, {year}, {time}
-      </Typography>
+      <>
+        {/* Feb 5th, 01:30 PM */}
+        {month} {day}{dateForm}, {startTime} - {endTime}
```

Comment:
Do you know if this is the standard? I've seen '~' and 'to' as well. Can you check which one we should use?